templates/default/opencast.service.erb

```diff
@@ -7,8 +7,8 @@ After=network.target
 After=remote-fs.target
 
 [Service]
-ExecStart=/<%= @opencast_root %>/bin/start-opencast server
-ExecStop=/<%= @opencast_root %>/bin/stop-opencast
+ExecStart<%= @opencast_root %>/bin/start-opencast server
```

Comment:
Is the "=" supposed to be missing here?

Reply:
yikes, no. i was just trying to sneak in removing the extra `/`.